SparkPost/2web2ui
# FE-895: Toggle component wrapper

### What Changed
 - Added Toggle component wrapper

### How To Test
 - Verify that toggles are working correctly
 - Verify that toggles do not have any visual defects on the regular app

### To Do
